### PR TITLE
Add ElfIDC to supported hosting providers

### PR DESCRIPTION
**Pull request by saltedfisholdxu**
add ElfIDC to support

### CODE REVIEW

File: _data/providers.json [hunk 1]
```diff
@@ -161,6 +161,11 @@
       "url": "https://eendhosting.nl/",
       "description_template": "default"
     },
+    {
+      "name": "ElfIDC",
+      "url": "https://www.elfidc.com/",
+      "description": "When placing an order, you can choose Paper/Spigot + Geyser, he will automatically install the Geyser plugin, and the player will connect through the Java IP and port, which is built-in."
```
Comment (onebeastchris) [inline _data/providers.json:167]:
```suggestion
      "description": "When placing an order, choose Paper/Spigot + Geyser, which will automatically install the Geyser plugin. Connect with the Java server's IP and port. Note: This provider only provides service in China."
```
I'd change the phrasing a bit - additionally, we historically added a note for country specific hosts (e.g. Russia). If this isn't the case, feel free to correct it

**Review by onebeastchris — CHANGES_REQUESTED** [review]
Looks good after these

File: _data/providers.json (hunk 1 above)
Comment (Konicai) [reply _data/providers.json:167]: They offer nodes that are based in Hong Kong, Los Angeles, and Japan: http://node.elfidc.com/

**Comment by saltedfisholdxu** [comment]
To be honest,we just support Minecraft server in China

**Comment by onebeastchris** [comment]
I see, I got a different impression since your site seems to be only in Chinese - if you offer services outside of China, then you don't need the additional note.

**Comment by saltedfisholdxu** [comment]
> I see, I got a different impression since your site seems to be only in Chinese - if you offer services outside of China, then you don't need the additional note.

When is the review likely to be completed and can it appear on the collaboration list?

**Comment by onebeastchris** [comment]
It's completed on our side, but there are requested changes [in a review i left](https://github.com/GeyserMC/GeyserWiki/pull/196#discussion_r1310509641) - once you address them, e.g. by applying the suggested change, this can get merged :)

**Comment by Kas-tle** [comment]
Please address the other changes with regards to grammar made by Chris (https://github.com/GeyserMC/GeyserWiki/pull/196#discussion_r1310509641). You can exclude the note regarding only providing services in mainland China since you have stated you also provide them elsewhere.

File: _data/providers.json (hunk 1 above)
Comment (saltedfisholdxu) [reply _data/providers.json]: I changed it and wish they can active

**Review by onebeastchris — APPROVED** [review]
Looks good to me, ty

**Comment by saltedfisholdxu** [comment]
> Looks good to me, ty

all right man,I just waiting for the merge PR right?

**Comment by onebeastchris** [comment]
Already merged it :)

**Comment by saltedfisholdxu** [comment]
> Already merged it :)

thanks  :>

**Pull request merged**